Release 1.0.0 THE NEXT WEB Responsive Design Project

pipe2442 opened the pull request:
Created navbar.

Added images and sections.

Made media queries elements.

Created footer.

Elements were styled and created for a responsive design.

Fixed stylelint errors

Validated HTML errors.

pipe2442 committed:
Created index file

pipe2442 committed:
Added nav bar for mobile view and started styling

mimipeshy committed:
added the article section

pipe2442 committed:
Added latest news section with images

pipe2442 committed:
Created latest news mobile section

mimipeshy committed:
added sponsor section

pipe2442 committed:
Added correcto background color to latest founding and correct h5 style

pipe2442 committed:
Fixed font wight for text in latest fundings and icons color

pipe2442 committed:
Added apps section

pipe2442 committed:
Fixed background image for parts

pipe2442 committed:
Added aps images with most popular text

pipe2442 committed:
Added list style and elements

mimipeshy committed:
added footer-section

mimipeshy committed:
added article-section

pipe2442 committed:
Added navbar to full screen section

mimipeshy committed:
Merge branch 'develop' into ft-navbar

pipe2442 committed:
Fixed media screen query

mimipeshy committed:
Merge branch 'develop' of https://github.com/pipe2442/responsivedesignProject into develop

mimipeshy committed:
worked on news section

mimipeshy committed:
fixed gap on widescreen

pipe2442 committed:
Added latest news section to full screen mode

pipe2442 committed:
Fixed correct width on full size for latest news

pipe2442 committed:
Fixed width on latest news section

pipe2442 committed:
made nav bar completely responsive now working on last news responsiveness

mimipeshy committed:
fixed mobile menu display to flex

pipe2442 committed:
Fixed latest news responsiveness

pipe2442 committed:
Merge branch 'develop' of https://github.com/pipe2442/responsivedesignProject into develop

pipe2442 committed:
Latest deal update with ordering grid

mimipeshy committed:
fixed main section using grid

mimipeshy committed:
fixed latest news section with grid

pipe2442 committed:
Corrected grid repeat section

pipe2442 committed:
Added margin for latest news section

mimipeshy committed:
fixed deals section desktop

pipe2442 committed:
Added stickler and linter file

pipe2442 committed:
Fixed margins and identation linter errors

pipe2442 committed:
Added comments to organize code elements and created media for middle tab view

pipe2442 committed:
Added responsiveness for nav bar for all views

pipe2442 committed:
Added wrap to latest found section

pipe2442 committed:
Final details to make navbar completely responsive

pipe2442 committed:
Fixed latest news responsiveness title

pipe2442 committed:
Sized icoons and added responsiveness to next section

pipe2442 committed:
Added fourth icon for latest founds

pipe2442 committed:
Fixed responisveness on nav bar menu

pipe2442 committed:
Fixed latest news padding and mobile margin

pipe2442 committed:
Fixed responsiveness on main image for web page

pipe2442 committed:
Fixed complete page responsiveness

pipe2442 committed:
Added latest deals images and cleaned html section

pipe2442 committed:
Fixed errors with html validator

pipe2442 committed:
Fixed documents title color

pipe2442 committed:
Fixed linter errors

mimipeshy committed:
fixed news section spacing

mimipeshy committed:
fixed spacing in latest news section

mimipeshy committed:
fixed spacing in sponsor section

mimipeshy committed:
fixed spacing in article section

mimipeshy committed:
fixed linter errors

pipe2442 committed:
Added screenshot

pipe2442 committed:
Update README.md

updated readme file

sumancrest0001 reviewed (commented):
**Status: changes required
Hello TEAM,**
You have done a great job to give the clone page a similar design layout as that of the original one. But We have to work a little bit more on the responsiveness of the page as it is not as responsive as the original one. Please compare to the [original page](https://thenextweb.com/).

Let's fix the following issues.

sumancrest0001 commented on assets/style.css:
```diff
@@ -0,0 +1,832 @@
+  opacity: 0.5;
+}
+
+.sponsor-assets {
+  padding: 10px 16px 10px 84px;
+  color: #000;
```
[Required] Let's use `rem`  instead of `px`. It will be very helpful in responsive design.
[Required] You are using larger font sizes than the original one. Let's try to follow the similar font-size, width, and weight as in the original one.
It should be :point_down: 
![Screenshot from 2020-06-02 15-27-53](https://user-images.githubusercontent.com/37747619/83507640-8b05c300-a4e8-11ea-9b91-734ecd843476.png)

But currently :point_down: 
![Screenshot from 2020-06-02 15-27-43](https://user-images.githubusercontent.com/37747619/83507687-9a850c00-a4e8-11ea-8cfd-d528a7f07947.png)

[Required] Align the main nav and secondary nav properly on the left side.

sumancrest0001 commented:
[Required] Let's give some padding to contents of the main article on the left and right sides.
![Screenshot from 2020-06-02 15-28-23](https://user-images.githubusercontent.com/37747619/83507950-f94a8580-a4e8-11ea-9562-85e676828868.png)

[Required] We have to align elements properly as in the original page compare Latest fonding sections 
in your project :point_down: 
![Screenshot from 2020-06-02 15-28-55](https://user-images.githubusercontent.com/37747619/83508315-783fbe00-a4e9-11ea-8737-01e754338791.png)

It should be :point_down: 
![Screenshot from 2020-06-02 15-29-09](https://user-images.githubusercontent.com/37747619/83508358-87bf0700-a4e9-11ea-8420-dc8f3b550bd0.png)

sumancrest0001 commented:
[Required] Images should have a similar height and width. currently, It looks like this :point_down: 
![Screenshot from 2020-06-02 15-29-51](https://user-images.githubusercontent.com/37747619/83508506-c228a400-a4e9-11ea-8dc2-dfc791b4945c.png)

It should be :point_down: 
![Screenshot from 2020-06-02 15-30-01](https://user-images.githubusercontent.com/37747619/83508548-cce33900-a4e9-11ea-9060-ff29f9260149.png)

[REQUIRED] The second footer section should be a one-liner. But it looks like this :point_down: 
![Screenshot from 2020-06-02 15-30-31](https://user-images.githubusercontent.com/37747619/83508660-f56b3300-a4e9-11ea-97a2-d10c4224ef8b.png)

It should be :point_down: 
![Screenshot from 2020-06-02 15-30-40](https://user-images.githubusercontent.com/37747619/83508695-0320b880-a4ea-11ea-97a6-485af749b78f.png)

sumancrest0001 reviewed (changes requested):
Please make the page responsive on medium and small screen referencing the original page. **Submit for another code review after making the needful changes.**

If you need any support or have questions regarding the review, feel free to DM me on slack **Suman Shrestha.
Happy coding !!!**

pipe2442 committed:
Fixed nav bar positioning, letter sizing latest funding rounds and generala web page responsiveness

pipe2442 committed:
Fixed footer in a single line

pipe2442 committed:
Added latest funding responsiveness for tab view

pipe2442 committed:
Added responsiveness for mobile view latest funding

pipe2442 committed:
Fixed information links for latest funds with correct info and images

pipe2442 committed:
Fixed titles news using rem and fixed for main view and responsive views

pipe2442 committed:
Added opacity to main titles

pipe2442 committed:
Fixed latest news responsive view for correct margin and paddings using rem

pipe2442 committed:
Fixed we heard mobile bar with correct pading using rem and mobile view aspect

pipe2442 committed:
Gave correct margin and padding to fixed image

pipe2442 committed:
Added icons to most popular section

pipe2442 committed:
Fixed linter erroors and validater index,html with html validator

pipe2442 committed:
Delete the next web screenshot.PNG

pipe2442 committed:
Add files via upload

pipe2442 committed:
Delete tnw screenshot.png

pipe2442 committed:
Add files via upload

pipe2442 committed:
Update README.md

pipe2442 commented:
Thanks for your help @sumancrest0001 all the changes were made!

peterrobert commented on index.html:
```diff
@@ -0,0 +1,753 @@
+                    </div>
+                    <div class="sponsor-details">
+                        <h3>Ahana</h3>
+                        <div class="sponsor-assets">
+                            <h4>$2.5M</h4>
+                            <h5>Seed Fund</h5>
+                        </div>
+                        <div class="sponsor-cards">
+                            <a href="#" class="sponsor-tag"><i class="fas fa-tag"><span> Information Technology</span></i></a>
+                            <a href="#" class="sponsor-tag"><i class="fas fa-briefcase">
+                                <span>GV</span></i></a>
+                            <div class="border">
+                                <p>Ahana gets seed funding <a href="#" class="faded-out">ZDNet</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="sponsor">
+                    <div class="sponsor-image">
+                        <img src="assets/images/sponsor3.jpeg" alt="sponsor image">
+                    </div>
+                    <div class="sponsor-details">
+                        <h3>Zeleros</h3>
+                        <div class="sponsor-assets">
+                            <h4>$7M</h4>
+                            <h5>Series A</h5>
+                        </div>
+                        <div class="sponsor-cards">
+                            <a href="#" class="sponsor-tag"><i class="fas fa-tag"><span> Design, +3</span></i></a>
+                            <a href="#" class="sponsor-tag"><i class="fas fa-briefcase">
+                                <span>Altran, +6</span></i></a>
+                            <div class="border">
+                                <p>Zeleros raises €7m <a href="#" class="faded-out">UKTN</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
```
There are still responsiveness issues in this section. Try and make the medium view (tablet view ) to look like the [original](https://thenextweb.com/). Your `.sponser` class should span the whole width at medium view check the [original](https://thenextweb.com/).

peterrobert commented on index.html:
```diff
@@ -0,0 +1,753 @@
+                <ul class="listOne">
+                    <li>
+                        <div class="sectionPopular8">
+                            <a href="#">
+                                <div class="news__title">
+                                    <h5 class="white no-margin mostPopular">Most popular</h5>
+                                    <h3 class="white no-margin">Chrome is testing dark mode for Google Search</h3>
+                                    <div class="news__author no-margin padding-top">
+                                        Mix 5 hours ago
+                                    </div>
+                                </div>
+                            </a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="latest">
+                            <div class="laTitles">
+                                <h5 class="no-margin">LATEST</h5>
+                                <h4 class="no-margin">Icon Rewind is a time machine for your phone’s app icons </h4>
+                            </div>
+                            <img src="assets/images/icon3.png" alt="icon 1">
+                        </div>
+                    </li>
+                </ul>
+                <ul class="listTwo">
+                    <li><a href="#" class="testing">YouTube now tells you to go to bed -- here's how to set it up</a>
+                    </li>
+                    <li><a href="#" class="testing">Google Maps can now highlight wheelchair-accesible busnesses</a>
+                    </li>
+                    <li><a href="#" class="testing">Facebook announces Workplace Rooms, basically Messenger Rooms for
+                            work</a>
+                    </li>
+                    <li><a href="#" class="testing">TikTok now gives you one-tap access to streaming app Resso</a></li>
+                    <li><a href="#" class="testing">Alook at how Jitsi became a 'secure' open-source alternative to
+                            Zoom</a>
+                    </li>
+                </ul>
+
+            </section>
+        </div>
```
There are still some responsiveness issues in this section. The images and the text is been hidden during the medium view. Try and center the images correctly and also the text should not be hidden. Try and make it like the [original](https://thenextweb.com/)

peterrobert reviewed (changes requested):
#changes required :recycle: 
Very great work in implementing the changes :1st_place_medal: However, there are still some few issues that are preventing me from approving this project. Please improve on the responsiveness issues stated in the comments.

Submit another review when you are done:smiley:

pipe2442 committed:
Fixed responsive view for tab and full screen

pipe2442 committed:
Merge branch 'develop' of https://github.com/pipe2442/responsivedesignProject into develop

pipe2442 committed:
Funding round completely fiex into fuill screen mid size and small screen view

pipe2442 committed:
Fixed popular section to stop making images to and text to hide with responsiveness

pipe2442 committed:
Fixed we heard text size for tab view and finished responsiveness for all sizings also fixed linter errors

pipe2442 committed:
Fixed final responsive design with margins and direction

pipe2442 commented:
Thanks for your review @peterrobert the changes that you requested have been made!

pipe2442 committed:
Update README.md

Updated live demo link.

pipe2442 committed:
Gave correct padding and margin for responsive view

pipe2442 committed:
Merge branch 'develop' of https://github.com/pipe2442/responsivedesignProject into develop

pipe2442 committed:
Update README.md

Updated live demo with correct link

tirthajyoti-ghosh reviewed (changes requested):
# Status: Changes required 🚨
## Great job implementing those changes 👌
But still, there are some issues that are preventing me from 👍 this project.

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+    <section class="news-section flex">
+        <div class="news__article1">
+            <div class="feature-main__top">
+                <img src="assets/images/tnw.svg" alt="" class="feature-article__logo" />
```
### The TNW logo in desktop mode is sticking to the corner and smaller than the original. The logo should be bigger and must not stick to the corner.
Right now 👇
![Annotation 2020-06-04 230308](https://user-images.githubusercontent.com/57726348/83791751-c99da800-a6b7-11ea-9eb8-d8b8c7a774ad.jpg)

It should be 👇
![Annotation 2020-06-04 230247](https://user-images.githubusercontent.com/57726348/83791799-d7ebc400-a6b7-11ea-988f-18c98ed16520.jpg)

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+    </header>
+
+    <section class="news-section flex">
+        <div class="news__article1">
```
### This div element -> `news__article1` should have some space on its left in desktop mode.

Right now 👇
![Annotation 2020-06-04 230308](https://user-images.githubusercontent.com/57726348/83792210-58122980-a6b8-11ea-986c-ee888fa38a69.jpg)

It should be 👇
![Annotation 2020-06-04 230247](https://user-images.githubusercontent.com/57726348/83792236-619b9180-a6b8-11ea-8dfd-b6338a1c0d1b.jpg)

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+            </div>
+        </div>
+
+        <section class="sponsor-section">
```
### The `sponsor-section` and `article-section` is shrinking in width when height changes in desktop mode. This  is not the case in the original site

Right now 👇
![Annotation 2020-06-04 231152](https://user-images.githubusercontent.com/57726348/83792596-e5557e00-a6b8-11ea-899a-d44485e3fc08.jpg)

It should be 👇
![Annotation 2020-06-04 231220](https://user-images.githubusercontent.com/57726348/83792687-01591f80-a6b9-11ea-8c85-84a191b8f06d.jpg)

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+    </section>
+
+    <main class="main">
+        <div class="latestNews">
```
The `latest news` section should align to 3 articles per row in medium screen mode. Right now it continues with the styles for the desktop screens. It should follow the the [original site](https://thenextweb.com/)

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+                <h3 class="news__title1">
+                    US Techs execs have dumped $3.6 billion in company stock this month
+                </h3>
+                <div class="news__author">
```
This and all div elements with this class -> `.news__author` changes position when height changes in small screen mode.

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+                        </div>
+                    </li>
+                </ul>
+                <ul class="listTwo">
```
This and all other -> `.listTwo`, texts inside them are too small to read. Increase their font size.

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,754 @@
+            </section>
+        </div>
+
+        <div class="latestNews">
```
In the "Latest Deals" section, each row should have 3 boxes in medium screen mode. Right now it continues with the styles for the desktop screens.

pipe2442 replied:
Thanks for your review Tirthajyoti, it has been really helpful. All the changes you requested were made!

tirthajyoti-ghosh reviewed (commented):
[OPTIONAL] Your README contains the description of a different project.
[OPTIONAL] Navbar menu links change color when hovered
[OPTIONAL] Social icons in footer changes styles when hovered

Apply these changes and request for another review.
Need any help, just shoot me a message in [Slack](https://microverse-students.slack.com/team/URB1MHXK6)

pipe2442 committed:
Made TNW logo bigger and fixed corner sticking

pipe2442 committed:
Gave correct margins and left space for news_article1 section in the desktop mode

pipe2442 committed:
Fixed correct shrinking for latest found part

pipe2442 committed:
Added correct shrink to article-section as the original site

pipe2442 committed:
Fixed section most popular letter

pipe2442 committed:
Fixed article section with correct letter sizing for responsivenss

pipe2442 committed:
Fixed letter sizing for latest funding section

pipe2442 committed:
Fixed responsive view for latest deals and latest news section and gave it correct width and margins

pipe2442 committed:
Fixed letters size in funding rounds for medium and full screen view

pipe2442 committed:
Gave correct letter sizing on mobile view for latest funding section

pipe2442 committed:
Added correct margin and padding for latest deals section also fixed responsive view

pipe2442 committed:
Fixed responsiveness for main news part and gave correct letter sizing to we heard title

pipe2442 committed:
Changed most popular images

pipe2442 committed:
Added correct sizing and marging details checked responsiveness and fixed linter errors

pipe2442 committed:
Add files via upload

pipe2442 committed:
Update README.md

Updated readme file.

pipe2442 committed:
sponsor and sponsorLast min width section

pipe2442 committed:
Merge branch 'develop' of https://github.com/pipe2442/responsivedesignProject into develop

pipe2442 committed:
Update README.md

Updated live demo link.

pipe2442 commented:
Release 1.0.4 THE NEXT WEB Responsive Design Project

- Fixed TNW logo with correct size and margins.
- Gave correct spacing to news__article1 section.
- Fixed sponsor-section and article-section shrinking in width.
- Give correct alignment to the Latest News section as the original site.
- Give the correct position and height to .news__author elements.
- Give correct size to .listTwo text elements.
- Give correct number of columns of each row of the "Latest Deals" section on the medium view.
- Fixed README.md file.

voscarmv commented:
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

voscarmv reviewed (approved):
# Excellent work applying the requested changes team! Project ✔️approved ✔️ 😄 👍 

**[OPTIONAL]** I strongly recommend adding a *Live Demo Link* on your `README.md` file! 👀 

Potential employers might be watching, and you definitely want them to be able to look at your work!

Other than that, it's all good 👍 

# Comments? Questions? Contact me on [Twitter](http://www.twitter.com/voscarmv) or [LinkedIn](http://www.linkedin.com/in/oscar-mier) 🧑‍💻 🗨️ 🧑‍💻 

## Happy Coding! 🧠 ⌨️ 💻

pipe2442 committed:
Update README.md

pipe2442 commented:
Thanks for your review @voscarmv the live demo was placed!
